packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -347,6 +350,10 @@ - (void)setZoomGesturesEnabled:(BOOL)enabled {
   _mapView.settings.zoomGestures = enabled;
 }
 
+- (void)setZoomControlsEnabled:(BOOL)enabled {
```

Comment:
This method doesn't seem to be called anywhere.

Reply:
And has been removed.